### ISSUE TITLE
🐛 [Bug Report] TBC - Zangarmarsh’s OpenPVP

### ISSUE THREAD

**Issue by Ischtaratu** [cmangos/issues#3768]
### Bug Details

![Screenshot 2024-09-18 041638](https://github.com/user-attachments/assets/402e8957-6665-406e-b69d-b22bdddfb6c5)

Horde flag is permanently displayed, the same when the twin spikes are inactive or the alliance has taken them over ... 

### Steps to Reproduce

1. Start server
2. go to twinpikes
3. 
4.

### Expected behavior

the horde flag should not be there in 2 of 3 possible situations 

### Suggested Workaround

n/a

### Core SHA1 Commit Hash

 Using commit hash(157110ebd0f9daab524cc5163fae030688fa6e2e) committed on 2024-09-15T19:19:40+02:00

### Database SHA1 Commit Hash

same day as core

### Operating System

W11 

### Client Version

2.4.3 (The Burning Crusade)

**Comment by Ischtaratu** [comment]
![Screenshot 2024-09-18 050926](https://github.com/user-attachments/assets/8c509013-0ff1-461d-b5d0-2295fe59a5b6)

Allianze has the twin spire but the flag is gone ...

**Comment by cpevors** [comment]
I, regrettably, cannot reproduce. See below. 

![WoWScrnShot_092224_214016](https://github.com/user-attachments/assets/22e34914-b466-46ab-a173-52656385cb7b)

**Comment by cpevors** [comment]
Spoke too soon. Can confirm. 

![WoWScrnShot_092224_220130](https://github.com/user-attachments/assets/dec308ea-ea76-4d63-9759-5574cdea56c2)

**Comment by maxkve** [comment]
Yes, there seems to be something strange with the conditions for displaying the flag gameobject perhaps? We've also been unable to capture the graveyard when both beacons were alliance controlled and graveyard horde controlled because the flag gameobject wasn't visible at all (except in GM mode).

When interacting with the flag in GM mode (with the battle standard), the graveyard was successfully captured and the alliance flag displayed for some time (2-3 minutes). At this point, the horde flag also became visible (like the screenshot in the original post).

**Core SHA1 Commit Hash**
6a85e79f1919935942e31c5b67b82d3b5c607105

**Database SHA1 Commit Hash**
b94c305a24abb5b72cd2a97320da4e9aecf386e9

**Operating System**
Ubuntu 22.04.5 LTS

**Client Version**
2.4.3 (The Burning Crusade)
